openshift-container-platform-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -2,42 +2,145 @@
 ## BEGINNING OF:
 ## SYSTEM AND SERVICES ACQUISITION
 ##
+## Reminder of "implementation_status" codes:
+##  - implementation_status: unsatisfied
+##  - implementation_status: unknown
+##  - implementation_status: planned
+##  - implementation_status: partial
+##  - implementation_status: complete
+##  - implementation_status: none
 
 - control_key: SA-1
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+    - key: a
+      text: |
+        Organizational processes to develop, document, and disseminate to
+        organization-defined personnel or roles a system and services
+        acquisition policy is outside the scope of
+        OpenShift configuration.
+    - key: a.1
+      text: |
+        Organizational processes to develop, document, and disseminate to
+        organization-defined personnel or roles a system and services
+        acquisition policy is outside the scope of
+        OpenShift configuration.
+    - key: a.2
+      text: |
+        Organizational processes to develop, document, and disseminate to
+        organization-defined personnel or roles a system and services
+        acquisition policy is outside the scope of
+        OpenShift configuration.
+    - key: b
+      text: |
+        Organizational review and updates to the current system and services
+        acquisition policy and procedures at an organization-defined frequency
+        is outside the scope of OpenShift configuration.
+    - key: b.1
+      text: |
+        Organizational review and updates to the current system and services
+        acquisition policy and procedures at an organization-defined frequency
+        is outside the scope of OpenShift configuration.
+    - key: b.2
+      text: |
+        Organizational review and updates to the current system and services
+        acquisition policy and procedures at an organization-defined frequency
+        is outside the scope of OpenShift configuration.
```

Comment:
Looking at the FedRAMP templates at https://www.fedramp.gov/assets/resources/templates/FedRAMP-SSP-High-Baseline-Template.docx, looks like only a SA-1(a b) level response is needed.